packages/ffe-account-selector-react/src/base-selector/BaseSelector.spec.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { BaseSelector, BaseSelectorProps } from './BaseSelector';
-import { act, fireEvent, render, screen } from '@testing-library/react';
```

Comment:
This test environment is not configured to use act. Trengtes nok før vi bumpade testrameverken. Kjører grønt slik det er men lager støy